step-templates/Remove-Hosts-File-Entry.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Id": "d8d02367-bda1-49ce-8e06-9fc34220843f",
+  "Name": "Windows - Remove Hosts File Entry",
+  "Description": "Remove given IP/host name entry exists in the hosts file.\n\nThis might be helpful in uninstall process.",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptBody": "$ip = $OctopusParameters['IP']\n$hostName = $OctopusParameters['HostName']\n\n$hostsPath = \"$env:windir\\System32\\drivers\\etc\\hosts\"\n$hosts = Get-Content $hostsPath\n$match = $hosts -match (\"^\\s*$ip\\s+$hostName\" -replace '\\.', '\\.')\n\nIf ($match) {\nwrite-host $hostsPath $hostsEntry \" exist, then it will be removed.\"\n}\nelse\n{\nwrite-host $hostsPath $hostsEntry \"not exist\"\nExit\n}\n\n$hostsEntry = \"$ip`t$hostName\"\n\n(Get-Content -Path $hostsPath) |\n    ForEach-Object {$_ -Replace \"$hostsEntry\"} |\n        Set-Content -Path $hostsPath -Verbose",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline"
+  },
+  "Parameters": [
+    {
+      "Id": "d9320ac1-f7cd-4895-9a9b-b2a35bc9bbe3",
+      "Name": "IP",
+      "Label": "IP Address",
+      "HelpText": "The IP address which the host name should resolve to, e.g. 127.0.0.1",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "129047cb-3e0c-4d83-a734-7d9bef6b5023",
+      "Name": "HostName",
+      "Label": "Host Name",
+      "HelpText": "The host name which should resolve to the given IP, e.g. www.mydomain.com",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    }
+  ],
+  "$Meta": {
+    "ExportedAt": "2021-08-04T19:43:51.043Z",
+    "OctopusVersion": "2020.4.6",
+    "Type": "ActionTemplate"
+  },
+  "LastModifiedBy": "Your GitHub Username",
```

Comment:
Hey @bilalmajali, one last request - can you update the file with your username here? It got dropped off with the fixed commit. 😄

Reply:
Hi @coryreid, my bad.. fixed.